cluster/gce/config-default.sh

```diff
@@ -298,7 +298,7 @@ if [ ${ENABLE_IP_ALIASES} = true ]; then
   # If we're using custom network, use the subnet we already create for it as the one for ip-alias.
   # Note that this means SUBNETWORK would override KUBE_GCE_IP_ALIAS_SUBNETWORK in case of custom network.
   if [[ "${CREATE_CUSTOM_NETWORK}" == true ]]; then
-    IP_ALIAS_SUBNETWORK="${SUBNETWORK:-IP_ALIAS_SUBNETWORK}"
+    IP_ALIAS_SUBNETWORK="${SUBNETWORK:-}"
```

Comment:
I don't understand it - shouldn't it be:
```
IP_ALIAS_SUBNETWORK="${SUBNETWORK:-${IP_ALIAS_SUBNETWORK}}"
```

Reply:
I don't think so. This should be the exact same subnet that we're creating in the custom network (which is `SUBNETWORK`).
And that *should be* non-empty in case of CREATE_CUSTOM_NETWORK (if it's not that's bad config)

Reply:
Ok - I see now. So then it should be:
```
IP_ALIAS_SUBNETWORK="${SUBNETWORK}"
```

instead - it's better to fail on config processing in that point, if we know this has to be non-empty.

Reply:
Changed it (though it should be a no-op as we're setting SUBNETWORK to non-empty value).

Reply:
It's no-op, but it sets clear expectations and it will fail immediately if the config is incorrect (instead of proceeding with strange empty subnet and failing sometime later...

Reply:
Makes sens